jdk/src/share/classes/sun/tools/attach/META-INF/services/openj9.tools.attach.diagnostics.spi.TargetDiagnosticsProvider

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2005, 2012, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This should have an IBM copyright, not an Oracle one. Use the one in `closed/make/CreateJars.gmk` as a guide.